rust/sealevel/environments/mainnet3/warp-routes/eclipseusdc/token-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "solanamainnet": {
+    "type": "collateral",
+    "decimals": 6,
```

Comment:
based on the solana and ethereum contracts:
- https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48
- https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v